blogs/8_berkeley_function_calling_leaderboard.html

```diff
@@ -111,7 +111,17 @@ <h4 class="text-center" style="margin: 0px;">
                         function calls with multiple programming languages and parallel and multiple function calls. We
                         also provide a specific debugging feature that when the provided function is not suitable for
                         your task, the model will output an “Error Message”.
-
+                    
+                    </p>
+                    <p>
+                        Quick Links:
+                        <ul>
+                            <li>Live Leaderboard: <a href="https://gorilla.cs.berkeley.edu/leaderboard.html">Website</a></li>
+                            <li>BFCL Evaluation Dataset: <a href="https://huggingface.co/datasets/gorilla-llm/Berkeley-Function-Calling-Leaderboard"> HuggingFace Dataset 🤗</a></li>
+                            <li>Gradio Demo: <a href="https://huggingface.co/spaces/gorilla-llm/gorilla-demo"> HuggingFace Space 🤗 </a></li>
```

Comment:
Can we link this to the "Try it Now" tab of the BFCL gradio? Since it is part of the OFv2 and not the original Gorilla models.